src/main/resources/align-aws-sdk-java-v2.json

```diff
@@ -0,0 +1,18 @@
+{
+  "align": [
+    {
+      "name": "align-aws-sdk-java-v2",
+      "group": "software.amazon.awssdk",
+      "includes": [".*"],
```

Comment:
You can leave as an empty array, it avoids the regex match.

Reply:
Done